docs/source/ruletypes.rst

```diff
@@ -642,6 +642,9 @@ Optional:
 
 ``smtp_host``: The SMTP host to use, defaults to localhost.
 
+``smtp_auth_file``: The path to a file which contains SMTP authentication credentials. It should be YAML formatted and contain
+two fields, ``user`` and ``password``. If this is not present, no authentication will be attempted.
+
```

Comment:
How is a user handled without a password? I presume by "no authentication", this will attempt anonymous SMTP?